### ISSUE TITLE
NTR: Carboxypeptidase N activity low in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8228]
New term request Carboxypeptidase N activity low in blood (elthat_130422154924):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000560
- label: Decreased circulating carboxypeptidase N activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:12560874
- def: Activity of the enzyme Carboxypeptidase N (arginine carboxypeptidase, EC 3.4.17.3) in the blood circulation below the lower limit of normal.
- comment: Carboxypeptidase N cleaves basic amino acid residues from the C terminal of peptides and proteins.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8228
- synonyms:
